system/datatypes/peer_reviewed_article.php

```diff
@@ -1462,6 +1463,17 @@ protected function createCommentDomElement(HyphaDomElement $comment, $review = f
 			return '<span><a href="' . $url . '" target="_blank">' . $url . '</a>' . $suffix.'</span>';
 		}, $commentHtml);
 		$commentHtml = nl2br($commentHtml);
+		if(strlen($commentHtml) > self::MAX_COMMENT_LENGTH){
+			$id = rand();
```

Comment:
This seems a bit fragile. I wonder if the javascript couldn't just find the right bits based on searching nearby the link? I.e. search for a certain class inside the parent element? IIRC we have jquery inluded everywhere, so that would probably make this easier?

Reply:
Done, vanilla js could do the trick as well :)

Reply:
Nice!